### PR TITLE
Updated conda reqs files for new packages

### PR DESCRIPTION
**Pull request by ianhelle**
Updated files in conda folder to align to new package versions and availability of packages from Microsoft conda repository

### CODE REVIEW

File: conda/conda-reqs.txt [hunk 1]
```diff
@@ -7,10 +7,11 @@ azure-keyvault-secrets>=4.0.0
 azure-kusto-data>=4.0.0, <=5.0.0
 azure-mgmt-compute>=4.6.2
 azure-mgmt-keyvault>=2.0.0
+azure-mgmt-monitor
```
Comment (petebryan) [inline conda/conda-reqs.txt:10]: Do we not want to use the version controls we had for pip here as well? I know azure package versioning has been a compatibility issue in the past.

File: conda/conda-reqs-pip.txt [hunk 2]
```diff
@@ -1,7 +1,8 @@
-azure-mgmt-monitor>=2.0.0
 azure-mgmt-resourcegraph>=8.0.0
-azure-mgmt-subscription>=3.0.0
-KqlmagicCustom[jupyter-basic,auth_code_clipboard]>=0.1.114.post22
+azure-monitor-query>=1.0.0, <=2.0.0
+# KqlmagicCustom[jupyter-basic,auth_code_clipboard]>=0.1.114.post22
```
Comment (petebryan) [inline conda/conda-reqs-pip.txt:3]: Whats the reason for commenting out this?

**Review by petebryan — APPROVED** [review]

File: conda/conda-reqs-pip.txt (hunk 2 above)
Comment (ianhelle) [reply conda/conda-reqs-pip.txt:3]: I'm hoping to remove dependency on Kqlmagic. It's still available as an option but unlike the other optional items in here it has a lot of dependencies. People will get a warning if they try to use one of the deprecated components that still use it and should have enough info to get it installed.

File: conda/conda-reqs.txt (hunk 1 above)
Comment (ianhelle) [reply conda/conda-reqs.txt:10]: Unfortunately, when I use standard versioning numbering it fails - this package in the Conda repo uses a date-based version string. Fortunately, conda handles versioning compatibility issues better than pip so it seems to resolve itself.

**Pull request merged**